src/librustdoc/config.rs

```diff
@@ -121,7 +121,7 @@ crate struct Options {
     /// For example, using ignore-foo to ignore running the doctest on any target that
     /// contains "foo" as a substring
     crate enable_per_target_ignores: bool,
-    /// Compile test but do not run them.
+    /// Do not run doctests, compile them if should_test is active.
```

Comment:
Isn't `should_test` active by default?

Reply:
I don't think so. It is activated when running `rustdoc --test ...`. By default it build the documentation without running or compiling tests